addons/backpacks/anarc210/ui/anarc210.hpp

```diff
@@ -174,8 +147,7 @@ class anarc210_radio_dialog
         action = "[TF_lr_dialog_radio, 1] call TFAR_fnc_setLrChannel;[""CH%1""] call TFAR_fnc_updateLRDialogToChannel;[TF_lr_dialog_radio, true] call TFAR_fnc_showRadioInfo;";
```

Comment:
we can turn this into a macro.
```
#define SET_CHANNEL_ACTION(chan) QUOTE([TF_lr_dialog_radio, chan] call TFAR_fnc_setLrChannel;[""CH%1""] call TFAR_fnc_updateLRDialogToChannel;[TF_lr_dialog_radio, true] call TFAR_fnc_showRadioInfo;)
action = SET_CHANNEL_ACTION(1);
```

Reply:
Maybe even put that into script_components in backpacks directory. Can reuse that over all our radios.